lib/matplotlib/texmanager.py

```diff
@@ -208,21 +178,16 @@ def __init__(self):
         cmd = [self.serif[1], self.sans_serif[1], self.monospace[1]]
         if self.font_family == 'cursive':
             cmd.append(self.cursive[1])
-        while '\\usepackage{type1cm}' in cmd:
-            cmd.remove('\\usepackage{type1cm}')
-        cmd = '\n'.join(cmd)
-        self._font_preamble = '\n'.join(['\\usepackage{type1cm}', cmd,
-                                         '\\usepackage{textcomp}'])
+        self._font_preamble = '\n'.join(
+            [r'\usepackage{type1cm}'] + cmd + [r'\usepackage{textcomp}'])
 
     def get_basefile(self, tex, fontsize, dpi=None):
         """
-        returns a filename based on a hash of the string, fontsize, and dpi
+        Returns a filename based on a hash of the string, fontsize, and dpi.
```

Comment:
Since you're changing them all, the first line of the docstring should be imperative (Return, not Returns).